config/environment.js

```diff
@@ -198,7 +198,7 @@ module.exports = function (environment) {
     },
     rootURL: '/',
     favicon: '/images/favicon.ico',
-    locationType: 'auto',
+    locationType: 'history',
```

Comment:
Would this change not affect browsers that do not support the HTML5 History API?

Reply:
No `auto` also defaults to `history` thats why `auto` option is getting removed in ember 5